src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrderListItems.js

```diff
@@ -0,0 +1,29 @@
+import MobileWorkingWorkOrderListItem from '../../WorkOrder/MobileWorkingWorkOrderListItem'
+
+export const MobileWorkingWorkOrderListItems = ({
+  workOrders,
+  currentUser,
+}) => {
+  const formatStatusText = (statusText) => {
+    const status = statusText.toLowerCase()
+
+    if (status === 'no access') return 'No access'
+    if (status === 'completed') return 'Completed'
+
+    return ''
+  }
+
+  if (workOrders === null || workOrders?.length === 0) {
+    return <></>
+  }
+
+  return workOrders.map((workOrder, index) => (
+    <MobileWorkingWorkOrderListItem
+      key={index}
```

Comment:
I suppose it's fine so long as no other components with `key={index}` are added to this page.

Reply:
The index only matters if it can be changed outside of react state